Zend/Optimizer/zend_optimizer.c

```diff
@@ -1426,10 +1425,11 @@ void zend_foreach_op_array(zend_script *script, zend_op_array_func_t func, void
 		zend_foreach_op_array_helper(op_array, func, context);
 	} ZEND_HASH_FOREACH_END();
 
-	ZEND_HASH_MAP_FOREACH_STR_KEY_PTR(&script->class_table, key, ce) {
-		if (ce->refcount > 1 && !zend_string_equals_ci(key, ce->name)) {
+	ZEND_HASH_MAP_FOREACH_VAL_IND(&script->class_table, zv) {
```

Comment:
You can probably use `ZEND_HASH_FOREACH_VAL` as we can't have indirect values here.

Reply:
Ah right indeed, my bad. I never really used these APIs before 😅

Reply:
No worries. I can't for the life of me memorize this stuff so I always have to look it up :smile: